### DISCUSSION TITLE
Icon management in shiny.fluent

### DISCUSSION THREAD

**Discussion by Alex33261** [Appsilon/shiny.fluent#121] [Q&A] [answered]
Hi, 

Please can you tell how we can manage icons while using shiny fluent ?

Thanks a lot, 

Alex

**Comment by jakubsob** [comment, marked as answer]
Hi @Alex33261,

with shiny.fluent you can still use `shiny::icon` and use Font Awesome or Glyphicon icons. When it comes to using Fabric icons (accessible using `shiny.fluent::Icon`) there is a [Microsoft Fabric Assets License Agreement](https://static2.sharepointonline.com/files/fabric/assets/microsoft_fabric_assets_license_agreement_10262017.pdf) which describes how those assets may be used.

**Comment by Alex33261** [comment]
Thank you very much Jakub.

Have a nice week-end,

Alex

Envoyé de mon iPhone
